picard/ui/tablebaseddialog.py

```diff
@@ -73,6 +74,14 @@ def prepare(self, headers):
         self.setRowCount(0)
         self.setSortingEnabled(False)
 
+    @throttle(1000)  # only emit resized signal once per second
+    def emit_resized(self):
+        self._parent_dialog.resized.emit()
+
+    def resizeEvent(self, event):
```

Comment:
Wouldn't it be easier to implement the `resizeEvent` on the `AlbumSearchDialog` directly?

Reply:
Well, since there was a `scrolled` signal already, I made both here since they are pretty similar (at it works as is).